rs/sns/governance/src/proposal.rs

```diff
@@ -1603,6 +1603,12 @@ async fn validate_and_render_register_extension(
 
 {extension_init}
 
+## Disclaimer
+
+Some Decentralized Exchanges do not provide the users with a slippage protection upon deposit.
+It means that the actual values that are going to be deposited to the exchange, could be significantly
+different from the asset ratio defined in the proposal.
```

Comment:
We should not only say that the deposit could be incomplete, but also that
* The resulting deposit may be abused by a 3rd party, e.g., the deposit may contribute to a liquidity pool with an unreasonable price, which may be susceptible to front-running or sandwich attacks.
* However, all tokens that cannot be deposited are sent back to the SNS treasury account.